Lib/test/test_plistlib.py

```diff
@@ -320,7 +320,6 @@ def test_tuple_members(self):
                     'second': [1, 2],
                     'third': [3, 4],
                 })
-                self.assertIsNot(pl2['first'], pl2['second'])
```

Comment:
Good catch! This test can fail in other implementations. It should be fixed in 3.6 too. Opened #4813 for this.